app/about/queries.js

```diff
@@ -1,6 +1,8 @@
 export async function getWorkdayData() {
-  const myHeaders = new Headers()
-  myHeaders.append("Content-Type", "application/json")
+  await new Promise(resolve => setTimeout(resolve, 3000)); // 3-second delay
```

Comment:
Is this 3-second delay necessary? What happened?

Reply:
that was for debugging so not necessary! but... i thought it was cute so i left it... but i took it out bc it makes our website seem slow and we have to be modern and lightning fast :)